src/binary_converter.py

```diff
@@ -0,0 +1,6 @@
+def main():
+    n = int(input("Insert first number: "))
+    print("The binary number is:", bin(n)[2:])
+
+
+main()
```

Comment:
Prova ad inserire questo check https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/